mkdocs.yml

```diff
@@ -4,5 +4,8 @@ theme: readthedocs
 repo_url: https://github.com/FakeItEasy/FakeItEasy.Analyzers
 extra_css: [css/extra-highlight.css]
 
-pages:
-- FakeItEasy Analyzer: index.md
\ No newline at end of file
+nav:
+- FakeItEasy Analyzer: index.md
+
+markdown_extensions:
+  - pymdownx.tilde
```

Comment:
with `pymdown-extensions` in the `requirements.txt`, activates our `~~`-based strikeout of the FakeItEasy0006 diagnostic:
![image](https://user-images.githubusercontent.com/3275797/70586162-61508c80-1b94-11ea-8905-131dcca5ca53.png)
instead of 
![image](https://user-images.githubusercontent.com/3275797/70586202-76c5b680-1b94-11ea-9d24-3a3a83db7316.png)

Reply:
👍